radar/infrastructure_config/portainer.yml

```diff
@@ -0,0 +1,14 @@
+version: 1
```

Comment:
Is this relevant when Kubernetes is on ADOPT?

Reply:
changing it to HOLD. We still struggle with k8s for our test environments. Portainer is used only in some of our test environments.